doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -149,8 +155,48 @@ class Frameworks(str, Enum):
+    ): "algorithms/vision/upscaling_hugging_face.rst",
+    (
+        Tasks.AUDIO_EMBEDDING,
+        Frameworks.TENSORFLOW,
+    ): "algorithms/audio/audioembedding_tensorflow.rst",
 }
 
+SAMPLE_NOTEBOOK_MODALITY_MAP = {
+    (Tasks.IC): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_image_classification/Amazon_JumpStart_Image_Classification.ipynb",
+    (Tasks.IC_EMBEDDING): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_image_embedding/Amazon_JumpStart_Image_Embedding.ipynb",
+    (Tasks.IS): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_instance_segmentation/Amazon_JumpStart_Instance_Segmentation.ipynb",
+    (Tasks.OD): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb",
+    (Tasks.OD1): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb",
+    (Tasks.SEMSEG): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation.ipynb",
+    (Tasks.TRANSLATION): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_machine_translation/Amazon_JumpStart_Machine_Translation.ipynb",
+    (Tasks.NER): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_named_entity_recognition/Amazon_JumpStart_Named_Entity_Recognition.ipynb",
+    (Tasks.EQA): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_question_answering/Amazon_JumpStart_Question_Answering.ipynb",
+    (Tasks.SPC): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb",
+    (Tasks.TC): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb",
+    (Tasks.TC_EMBEDDING): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_embedding/Amazon_JumpStart_Text_Embedding.ipynb",
+    (Tasks.TEXT_GENERATION): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_generation/Amazon_JumpStart_Text_Generation.ipynb",
+    (Tasks.SUMMARIZATION): "https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart_text_summarization/Amazon_JumpStart_Text_Summarization.ipynb",
+    # TODO: Find notebook link for  this task
+    (Tasks.ZSTC): "https://www.google.com/",
```

Comment:
maybe we should use `amazon.com` as placeholder ;)

Reply:
How does rickrolling our customers sound? I can put in that url lol